backend/ddang/src/main/java/com/ddang/ddang/chat/presentation/dto/CreateMessageRequest.java

```diff
@@ -0,0 +1,13 @@
+package com.ddang.ddang.chat.presentation.dto;
+
+import jakarta.validation.constraints.NotNull;
+
+public record CreateMessageRequest(
+        @NotNull(message = "메시지 발신자 아이디가 입력되지 않았습니다.")
+        Long writerId,
+        @NotNull(message = "메시지 수신자 아이디가 입력되지 않았습니다.")
+        Long receiverId,
+        @NotNull(message = "메시지 내용이 입력되지 않았습니다.")
+        String contents
```

Comment:
`@NotNull`과 같은 어노테이션 사용 시 개행을 주기로 했었습니다!

```suggestion
        @NotNull(message = "메시지 발신자 아이디가 입력되지 않았습니다.")
        Long writerId,
        
        @NotNull(message = "메시지 수신자 아이디가 입력되지 않았습니다.")
        Long receiverId,
        
        @NotNull(message = "메시지 내용이 입력되지 않았습니다.")
        String contents
```

Reply:
수정해야겠다고 생각했었는데 그대로 제출했네요..... 제송함니다... 🙇🏻‍♀️